dist/sse/event-source.js

```diff
@@ -0,0 +1,108 @@
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
+    function adopt(value) { return value instanceof P ? value : new P(function (resolve) { resolve(value); }); }
+    return new (P || (P = Promise))(function (resolve, reject) {
+        function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
+        function rejected(value) { try { step(generator["throw"](value)); } catch (e) { reject(e); } }
+        function step(result) { result.done ? resolve(result.value) : adopt(result.value).then(fulfilled, rejected); }
+        step((generator = generator.apply(thisArg, _arguments || [])).next());
+    });
+};
+export class EventSource {
+    constructor(url, headers) {
+        this.properties = {};
+        this.url = url;
+        this.headers = headers;
+    }
+    onOpen(listener) {
+        this.openListener = listener;
+    }
+    onProperties(listener) {
+        this.propertiesListener = listener;
+    }
+    onMessage(listener) {
+        this.messageListener = listener;
+    }
+    onError(listener) {
+        this.errorListener = listener;
+    }
+    onClose(listener) {
+        this.closeListener = listener;
+    }
+    connect(data) {
+        return __awaiter(this, void 0, void 0, function* () {
+            var _a, _b, _c, _d, _e, _f;
```

Comment:
```suggestion
            let _a, _b, _c, _d, _e, _f;
```

Reply:
I cannot change it as it is autogenerated